evm/src/fuzz/invariant/executor.rs

```diff
@@ -0,0 +1,290 @@
+use super::{
```

Comment:
I'm not a big fan of dividing the impl of the InvariantExecutor across multiple files, makes it harder to review

proposing to merge all `impl InvariantExecutor` into a single file